ui/src/telegrafs/components/Collectors.tsx

```diff
@@ -115,7 +115,7 @@ class Collectors extends PureComponent<Props, State> {
               <GetResources resource={ResourceType.Labels}>
                 <FilterList<Telegraf>
                   searchTerm={searchTerm}
-                  searchKeys={['plugins.0.config.bucket', 'labels[].name']}
+                  searchKeys={['plugins.0.config.bucket', 'name']}
```

Comment:
this is great! Could you make an issue to add filtering by label name as well?

Reply:
@ebb-tide I'd be happy to!